content/tutorial/exploration.md

```diff
@@ -81,13 +56,37 @@ setInterval(gameLoop, 1000)
 Straight walk will keep your character moving in a straight line until it hits a wall.
 Then it will move in a random direction. This will make your character explore the game world in a more structured way.
 
+## Cardinal direction movement
+
+```js
+
+
+function moveInDirection(dir) {
+  const directions = {
+    north: [ 0,-1], 
+    east : [ 1, 0],
+    south: [ 0, 1],
+    west : [-1, 0],
+  };
+  let movePoint = {x: dw.c.x, y: dw.c.y};
+  movePoint.x += directions[dir][0];
+  movepoint.y += directions[dir][1];
+  dw.move(movePoint.x, movePoint.y);
+}
+
+let direction = "north"
+setInterval(() => {
+  moveInDirection(direction);
+}, 1000);
+```
+
 ## Detecting Walls and Holes
 
 You can access the tile you are standing on like this:
 
 ```js
-dw.getTerrainAt(dw.character.x, dw.character.y, dw.character.z) // This will return the number of the tile you are standing in
```

Comment:
I'm a bit against shorting here, if unfamiliar `dw.character` is more understandable than `dw.c`